js/okx.js

```diff
@@ -1108,24 +1107,24 @@ module.exports = class okx extends Exchange {
             'optionType': optionType,
             'precision': {
                 'amount': this.safeNumber (market, 'lotSz'),
-                'price': precisionPrice,
+                'price': this.parseNumber (tickSize),
             },
             'limits': {
                 'leverage': {
                     'min': this.parseNumber ('1'),
                     'max': this.parseNumber (maxLeverage),
                 },
                 'amount': {
-                    'min': minAmount,
-                    'max': undefined,
+                    'min': this.safeNumber (market, 'minSz'),
+                    'max': this.safeNumber (market, 'maxLmtSz'),
```

Comment:
@samgermain we have a lot of different `maxSizes` shouldn't we keep the maximum among all? `max(value1, value2, value2)` ?
![image](https://user-images.githubusercontent.com/43336371/218695067-53e1e22e-3d2a-40c5-a9c8-651c2035290a.png)

Reply:
> @samgermain we have a lot of different `maxSizes` shouldn't we keep the maximum among all? `max(value1, value2, value2)` ? ![image](https://user-images.githubusercontent.com/43336371/218695067-53e1e22e-3d2a-40c5-a9c8-651c2035290a.png)

I actually feel like it might be better as `undefined`, otherwise it might be misleading